README.template.md

```diff
@@ -0,0 +1,37 @@
+{{ ossHeader }}
+
+# Momento Swift SDK
+
+To get started with Momento you will need a Momento Auth Token. You can get one from the [Momento Console](https://console.gomomento.com).
```

Comment:
should we use "api key" instead of "auth token"?

Reply:
Yes, yes we should :-) Thanks.